src/components/dataset/DatasetScreen.js

```diff
@@ -86,10 +86,13 @@ class DatasetScreen extends Component {
 
   renderDatasetInformation = () => {
     const { dataset, t } = this.props;
-    const content = dataset?.get('content') || '';
-    const parsedContent = JSON.parse(content)?.data;
+    const content = dataset?.get('content');
+    if (!content) {
+      return null;
+    }
+    const parsedContent = JSON.parse(content)?.data || {};
 
-    const { actions = [] } = parsedContent;
+    const actions = parsedContent?.actions.filter((action) => action) || [];
```

Comment:
If the actions are deleted, then there is an error on the `filter` call. Should be `parsedContent?.actions?.filter`